kafka/config/server.properties

```diff
@@ -114,6 +114,6 @@ zookeeper.connect=ZOOKEEPER_SERVERS
 
 
 # Timeout in ms for connecting to zookeeper
-zookeeper.connection.timeout.ms=6000
+zookeeper.connection.timeout.ms=60000
```

Comment:
I don't think this change makes any sense.

Reply:
ok, I will fix